src/components/ebay-listbox-button/README.md

```diff
@@ -33,6 +33,7 @@ Name | Type | Stateful | Required | Description
 `name` | String | No | Yes | used for the `name` attribute of the native `<select>`
 `selected` | Number | Yes | n/a | allows you to set the selected index option to `selected`
 `borderless` | Boolean | No | No | whether button has borders
+`fluid` | Boolean | No | No | wether listbox width is 100%
```

Comment:
Minor, but a typo on 'wether' (unless it's some americanism I'm unfamiliar with).

Reply:
No, that's 100% a typo.